backend/endpoints/train_endpoints.py

```diff
@@ -70,8 +75,35 @@ def tabular_run():
         )
 
         try:
-            createTrainspaceData(tabular_data)
+            create_dynamo_item_from_obj("trainspace", tabular_data)
             print(id)
+
+            async def run():
+                async with aiohttp.ClientSession(json_serialize=json.dumps) as session:
+                    async with session.post(
```

Comment:
if we need to use `aiohttp` in other places in our backend, do you think that we need to have some sort of generalized way to invoke the endpoint instead of duplicating the boilerplate all over. 

Generally, when you need to talk from one service to another, you'll need to setup something called a data client of some sort. That entity acts as the centralized place for one service to talk to another (and you preserve abstraction and minimize coupling). What this code demonstrates is the use of what's called an RPC call (Remote Procedure Call). This code in our API is making a request to another service and awaiting the result

Reply:
Would like to highlight this code as a good learning example for others on DLP to understand

Reply:
Yeah creating a class or set of functions that handles communicating to the training microservice could be a good idea